feed_amalgamator/helpers/mastodon_oauth_interface.py

```diff
@@ -200,10 +200,10 @@ def add_domain_to_database(self, domain_name):
         token_url = "https://" + domain_name + "/oauth/token"
         # TODO: Will need to refactor the code below into several functions
         payload = {
-            'client_name': 'Test Application',
-            'redirect_uris': 'urn:ietf:wg:oauth:2.0:oob',
-            'scopes': 'read write push',
-            'website': 'https://myapp.example'
+            "client_name": "Feed Amalgamator",
+            "redirect_uris": self.REDIRECT_URI,
+            "scopes": "read write push",
+            "website": "http://127.0.0.1:5000",
```

Comment:
@samarth9201 can you please explain what this payload is used for and what the parameter "website" refers to?

Reply:
"website" is a required parameter for the API call. It is not used after registering the application. The most important fields in this payload is "redirect_uris" and "scopes".